Formula/rpm.rb

```diff
@@ -27,54 +12,46 @@ class Rpm < Formula
     sha256 "3a276b6a3f7273f8f88effbb057e8d65fc53fa9e812054951bfdd5ce618f54ac" => :yosemite
   end
 
-  depends_on "rpm2cpio" => :build
+  depends_on "pkg-config" => :run
   depends_on "berkeley-db"
+  depends_on "gettext"
+  depends_on "libarchive"
   depends_on "libmagic"
+  depends_on "lua"
+  depends_on "openssl@1.1"
   depends_on "popt"
-  depends_on "libtasn1"
-  depends_on "gettext"
   depends_on "xz"
-  depends_on "ossp-uuid"
+  depends_on "zstd"
 
   def install
     # only rpm should go into HOMEBREW_CELLAR, not rpms built
-    inreplace "macros/macros.in", "@prefix@", HOMEBREW_PREFIX
+    inreplace ["macros.in", "platform.in"], "@prefix@", HOMEBREW_PREFIX
     args = %W[
```

Comment:
Since there are no options all of these arguments can be inlined after configure.

```
    system "./configure", "--disable-dependency-tracking",
                          "--disable-silent-rules",
                          "--prefix=#{prefix}",
                          "--localstatedir=#{var}",
                          "--sharedstatedir=#{var}/lib",
                          "--sysconfdir=#{etc}",
                          "--with-path-magic=#{HOMEBREW_PREFIX}/share/misc/magic",
                          "--enable-nls",
                          "--disable-plugins",
                          "--with-external-db",
                          "--with-crypto=openssl",
                          "--without-apidocs",
                          "--with-vendor=homebrew"
```
Note I've added `--disable-silent-rules` and removed `--with-libiconv-prefix=/usr`, which isn't needed.

Also, are you sure we still need
```
--with-path-magic=#{HOMEBREW_PREFIX}/share/misc/magic
```
?

Reply:
I believe it's still needed, but I'll push the other requested change.